packages/cf-builder-table/test/TableBuilder.js

```diff
@@ -101,6 +110,6 @@ test('should support flashes', () => {
     </Provider>
   );
   store.dispatch(tableActions.flashRow('test-table', '1', 'success'));
-  const row = wrapper.find(TableBody).find(TableRow).at(0);
-  expect(row.prop('type')).toBe('success');
+  const row = wrapper.find('tbody').find('tr').at(0);
```

Comment:
Enzyme was giving me this error:

```
Method “props” is only meant to be run on a single node. 0 found instead.
```

Not sure why so I'm testing against the markup.